src/pages/SearchPage.js

```diff
@@ -33,16 +33,19 @@ const propTypes = {
 
     /** Whether we are searching for reports in the server */
     isSearchingForReports: PropTypes.bool,
+
+    navigation: PropTypes.shape({}),
```

Comment:
Can you please add a comment here to adhere with the style rules. Thanks!

Reply:
ideally noting this is for automated tests

Reply:
sure, will do